# NeighborsArray.msg
Header header
Neighbors[] neighbors_array
